project/AppDependencies.scala

```diff
@@ -1,31 +1,31 @@
-import sbt._
+import sbt.*
 
 object AppDependencies {
   import play.core.PlayVersion
 
-  val bootstrapVersion = "7.13.0"
+  val bootstrapVersion = "7.20.0"
 
-  val compile = Seq(
+  val compile: Seq[ModuleID] = Seq(
     play.sbt.PlayImport.ws,
     "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-28"             % "0.74.0",
```

Comment:
most probably this can be updated as the repository in this repo is super simple, it can be tested wit AT as it only has basic set/get methods for user answers, but it is ok if you want to leave it for next time as well

Reply:
prefer to leave it, we have separate tickets for mongo